web/ui/react-app/src/pages/Config.tsx

```diff
@@ -1,6 +1,46 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { RouteComponentProps } from '@reach/router';
+import { Alert, Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import CopyToClipboard from 'react-copy-to-clipboard';
 
-const Config: FC<RouteComponentProps> = () => <div>Config page</div>
+import './Config.css';
+
+const Config: FC<RouteComponentProps> = () => {
+  const [config, setConfig] = useState(null);
+  const [error, setError] = useState("");
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
```

Comment:
You have to specify empty array as a second arg of useEffect in order to prevent API calls on every component update.
Thus it will act as componentDidMount

Reply:
Ah cool, didn't know about this. This PR was my first time ever using hooks :)